casepro/cases/management/commands/bootstrap_dev.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import unicode_literals
```

Comment:
This must be a merge problem. I'll refresh with the latest and see what happens.